rdbms/src/main/java/org/spine3/server/storage/jdbc/JdbcStorageFactory.java

```diff
@@ -51,19 +46,20 @@
  * @author Alexander Litus
  * @author Andrey Lavrov
  */
-public class JdbcStorageFactory implements StorageFactory {
+public class JdbcStorageFactory<I> implements StorageFactory {
```

Comment:
`I` param description should be added to JavaDoc section. 
See `org.spine3.server.entity.EntityRepository` for doc sample.